feconf.py

```diff
@@ -112,6 +112,10 @@
     TRAINING_JOB_STATUS_FAILED: [TRAINING_JOB_STATUS_NEW]
 }
 
+# The trivial interaction ids which don't need the true instance of
```

Comment:
Interaction IDs for which answer details cannot be solicited.

Reply:
Done